src/lidar/read_hokuyo_lidar.cpp

```diff
@@ -0,0 +1,37 @@
+
+#include "PointCloudProcessing.h"
+#include "PointGenerator.h"
+#include "URGLidar.h"
+#include "../simulator/utils.h"
+
+bool lidar_initialized = false;
+URGLidar urg_lidar;
+
+points_t readLidarScan() {
```

Comment:
If this is the implementation of `readLidarScan()` in `world_interface.h`, then `world_interface.h` should probably be included in this file.